protocol/test/amo/oud/oud-token.ts

```diff
@@ -0,0 +1,210 @@
+import { expect } from "chai";
+import { OudToken, OudToken__factory } from "../../../typechain";
+// import { recoverToken, shouldRevertNotOwner, testErc20Permit } from "../../helpers";
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { BigNumberish, Signer, TypedDataDomain, TypedDataField } from "ethers";
+import { ethers } from "hardhat";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import { splitSignature } from "ethers/lib/utils";
+import { blockTimestamp } from "../../helpers";
+ 
+describe("Mintable Token", async () => {
```

Comment:
Update the name here

Reply:
fixed in [ac5905d](https://github.com/TempleDAO/temple/pull/760/commits/ac5905d0d1b244eee387df8ce2f0d382287efcde)